src/showcase/Showcase.story.mdx

```diff
@@ -0,0 +1,34 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs/blocks'
+
+import { Paragraph } from '../paragraph'
+import { Showcase } from './index'
+
+<Meta title="Widgets/Showcase" />
+
+# Showcase
+
+<Canvas>
+  <Story name="Default">
+    <Showcase>
```

Comment:
Not sure about the a11y of this component (right now, it's not too good). Having moving UI pieces is usually annoying for impaired users (since it's hard to make it accessible) but I guess I can look somewhere else.

This remarks apply for SEO crawlers a11y too: ~~They will only have access to the first item in the showcase when crawling this page.~~ It's all opacity based so it should be okish for SEO. (https://stackoverflow.com/questions/3593158/does-google-index-pages-with-opacity0-or-hidden-or-displaynone)

Reply:
This is not purely for SEO, rather for SEA. Nevertheless, for another component we checked with our expert to see if "display: none" was to be avoided and they said it was OK, as far as all the content is there on the first render. Further improvements might come later.

As for A11Y, I would agree but right now we're only replacing a legacy page "as is". I do hope that more time is spent on designing a whole new project on the matter of SEA pages but it's not going to happen soon IMO.